### PR TITLE
Add MDS Metrics API spec

### PR DESCRIPTION
**Pull request by whereissean**
# MDS Pull Request

## Explain pull request

This PR represents a proposal for a standard interface for historical MDS metrics and data aggregation.

## Is this a breaking change

* No, not breaking

## Impacted Spec

* `agency`
* `provider`

## Additional context

This PR is related to the MDS Metrics Issue #485.

### CODE REVIEW

File: metrics/README.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+`POST /metrics`
+
+Supports querying one or more metrics with the following parameters.
+
+### Parameters
+
+| Name            | Type     | Required | Comments                                                          |
+| --------------- | -------- | -------- | ----------------------------------------------------------------- |
+| `metrics`       | string[] | Yes      | list of metrics to return. [See metric names](core_metrics.md)    |
+| `interval`      | duration | Yes      | Duration for metrics intervals.                                   |
```
Comment (jfh01) [inline metrics/README.md:108]:
```suggestion
| `interval`      | duration | Yes      | ISO 8601 formatted duration for metrics intervals.                                   |
```

**Comment by schnuerle** [comment]
Note that for 1.1.0 we have [merged with #582](https://github.com/openmobilityfoundation/mobility-data-specification/pull/582) the new [Geography API](https://github.com/openmobilityfoundation/mobility-data-specification/tree/dev/geography) to the 'dev' branch. Please update this pull request with the latest code, resolve any conflicts, and make references to the Geography API where appropriate, e.g. with UUIDs. 

We will be discussing Metrics at this week's [Working Group meeting](https://github.com/openmobilityfoundation/mobility-data-specification/wiki/Web-conference-notes,-2020.10.15-(Joint-Working-Group-City-Services)), so if available please come prepared to talk about your latest updates and ideas.

**Comment by schnuerle** [comment]
Leaving a comment here from #123 (now closed in favor of this PR).  Wanted info on subscribers and low_income riders.  

 `membership_type` |
|--------------|
| subscriber             |
| subscriber_low_income  |
| single_ride            |
| single_ride_low_income |

**Comment by schnuerle** [comment]
Leaving a comment here from #76 (now closed in favor of this PR). Wanted info on access methods.  Could be part of #569 too. 

| `access_method` | meaning                                                                        |
|-----------------|--------------------------------------------------------------------------------|
| `qr_code`       | The user scanned a QR code with their mobile device                            |
| `keypad`        | The user entered an identification number on a keypad                          |
| `transit_card`  | The user used a transit card to unlock the vehicle                             |
| `rfid`          | The user accessed the vehicle with an RFID card or fob that is not a transit card |
| `mobile_id`     | The user entered the ID of the device on their mobile phone                    |
| `other_mobile`  | The user used a mobile phone, but did not scan a QR code or enter a vehicle ID |
| `credit_card_pay`   | The user swiped/inserted/tapped their credit card to pay for the ride      |
| `credit_card_id`    | The user swiped/inserted/tapped their credit card to identify themselves, but their card was not charged an additional amount |
| `cash_attendant` | The user paid an attendant or operator cash to use the vehicle                |
| `cash_automated` | The user paid cash using an automated system                                  |
| `call_center`    | The user accessed the vehicle by calling a call center operated by humans     |
| `ivr`            | An interactive voice response system was used to access the vehicle           |
| `sms`            | An SMS system was used to access the vehicle                                  |
| `no_security`    | The vehicles are unsecured and can be used by anyone without charge           |

**Comment by schnuerle** [comment]
I've made a feature branch called '[feature-metrics](https://github.com/openmobilityfoundation/mobility-data-specification/tree/feature-metrics)' to start pulling all the Metrics related work together and do PRs against. Updated this PR to point to that instead of 'dev'.

**Comment by schnuerle** [comment]
Closing this since the entire PR was incorporated into #487 for review and before merger to the Metrics feature branch.

**Pull request closed without merging**